bitnami/wavefront-adapter-for-istio/templates/NOTES.txt

```diff
@@ -14,10 +34,11 @@ APP VERSION: {{ .Chart.AppVersion }}
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
            You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ template "common.names.fullname" . }}
     export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "common.names.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-    echo http://$SERVICE_IP:{{ .Values.service.port }}
+    echo http://$SERVICE_IP:{{coalesce .Values.service.ports.grpc service.port}}
 {{- else if contains "ClusterIP" .Values.service.type }}
-    echo "The renconciler is available at http://127.0.0.1:{{ .Values.service.port }}"
-    kubectl port-forward svc/{{ template "common.names.fullname" . }} {{ .Values.service.port }}:{{ .Values.service.port }} &
+    echo "The renconciler is available at http://127.0.0.1:{{coalesce .Values.service.ports.grpc service.port}}"
+    kubectl port-forward svc/{{ template "common.names.fullname" . }} {{coalesce .Values.service.ports.grpc service.port}}:{{coalesce .Values.service.ports.grpc service.port}} &
+{{- end }}
```

Comment:
I may have induced this typo by accident, but it should be `.Values.service`.

Check helm tests to see the chart are failing because of this